blog/2023-06-22-openfeature-adoption-story.md

```diff
@@ -0,0 +1,129 @@
+---
+slug: 'openfeature-adoption-story'
+title: 'OpenFeature Adoption Story'
+date: 2023-06-22
+authors: [liran2000]
+description: "OpenFeature is an open standard for feature-flagging. This is an adoption story."
+tags: [client, sdk, adoption]
+draft: false
+---
+
+Our team recently needed to use one of the largest feature management cloud services.
```

Comment:
Can we mention which one?

Reply:
I don't think that's necessary. One of the primary values of OpenFeature is vendor neutrality. Leaving this as-is helps emphasize that.